src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/__tests__/features/form/data/submitFormDataSagas.test.ts

```diff
@@ -53,7 +53,7 @@ describe('submitFormDataSagas', () => {
       formData: getFormDataStateMock({
         formData: {
           field1: 'value1',
-          field2: 123,
+          field2: '123',
```

Comment:
Why the change from number to string here - are numbers acutally not valid here?
I dont know this area too well..

Reply:
See https://github.com/Altinn/altinn-studio/pull/7394. We need to handle all data as strings in fronted. (html `<input` does that anyway.)

Reply:
Makes sense, thanks for the clarification :)